sdk/servicebus/service-bus/rollup.base.config.js

```diff
@@ -86,7 +94,7 @@ export function nodeConfig({ test = false, production = false } = {}) {
 export function browserConfig({ test = false, production = false } = {}) {
   const baseConfig = {
     input: input,
-    external: [],
+    external: ["./aadUtils", "./utils/aadUtils"],
```

Comment:
This isn't a good approach because dependents won't use this rollup config and will need to know how to handle these files if they're bundling for browser.

Instead, create .browser.ts alternatives and map them using the package.json's browser field. amqp-common demonstrates this approach.

Reply:
Since this is for in just tests, would it affect dependents?
I've moved it to within test specific rollup, would this help?

On a side note, I've just tried adding the two files and entered mapping in package.json's browser just like amqp-common but it didn't make any difference. Presumably as it affects only dependents and not tests while packaging? The tests are making a direct reference to the source so this isn't working

Reply:
I've cleaned it up some more to externalize "@azure/ms-rest-nodeauth" instead similar to Node test rollup config.

Reply:
The mapping should work internally as well. If you'd like to get that working we can maybe sit together/use liveshare and we can figure it out. Am ok with externalizing @azure/ms-rest-nodeauth in test environments, but does a non-test build even work since ms-rest-nodeauth won't be external and will thus get bundled?

Reply:
Sure, thanks will sync up on it and maybe we can look into that when looking into #2556 

Right, other functionality that depend on it won't work - moved it out to general browser rollup.
Other than that, I just ran the sample against main build - simple send works.

Reply:
I looked at this today, and ms-rest-nodeauth is only used in two cases:

1. It provides type definitions needed at compile time for the library itself
2. It is used inside `if (isNode)` blocks in the tests

Therefore it seems safe to external out.